io.github.Soundux.yml

```diff
@@ -0,0 +1,31 @@
+app-id: io.github.Soundux
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: soundux
+finish-args:
+  - --device=dri
+  - --socket=pulseaudio
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=xdg-music:ro
+  - --filesystem=xdg-download:ro
+  - --filesystem=xdg-documents:ro
+  - --filesystem=/media:ro
+  - --filesystem=/run/media:ro
+  - --filesystem=/media:ro
+  - --filesystem=~/.config/Soundux
```

Comment:
```suggestion
```

This is something that should generally work out of the box. Flatpak sets `XDG_CONFIG_HOME` to a directory writable by default.

Reply:
I tried it but it doesn't work without it

Reply:
I think this is because we save the config under `$HOME/.config/Soundux`

Reply:
I adjust it to use `$XDG_CONFIG_HOME` if set